pom.xml

```diff
@@ -68,7 +68,11 @@
             <artifactId>liquibase-core</artifactId>
             <version>${liquibase.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>8.0.1.Final</version>
```

Comment:
Why this dependency is required? How it is related to exception handling?

Reply:
I need those for validation